index.html

```diff
@@ -1,1589 +1,1537 @@
-                        <li>Append the value of <var>prefix definition</var> to <var>element prefixes list</var>.
-                    </ol>
-                </ol>
-            </ol>
-            <li>Return the value of <var>default namespace attr value</var>.
-        </ol>
+  <p>Conformance requirements phrased as algorithms or specific steps may be implemented in any
+  manner, so long as the end result is equivalent. (In particular, the algorithms defined in this
+  specification are intended to be easy to follow, and not intended to be performant.)
 
-        <p>To <dfn title="concept-generate-prefix">generate a prefix</dfn> given a
-            <a title="concept-namespace-prefix-map">namespace prefix map</a> <var>map</var>, a
-            string <var>new namespace</var>, and a reference to a
-            <a title="concept-generated-prefix">generated namespace prefix index</a> <var>prefix
-            index</var>, the user agent must run the following steps:
-        <ol>
-            <li>Let <var>generated prefix</var> be the concatenation of the string "<code>ns</code>" and
-                the current numerical value of <var>prefix index</var>.
-            <li>Let the value of <var>prefix index</var> be incremented by one.
-            <li>Append to <var>map</var> a new key <var>new namespace</var> whose key value is the
-                <var>generated prefix</var>.
-            <li>Return the value of <var>generated prefix</var>.
-        </ol>
+  <p>User agents may impose implementation-specific limits on otherwise unconstrained inputs, e.g.
+  to prevent denial of service attacks, to guard against running out of memory, or to work around
+  platform-specific limitations.
 
-        <p>The <dfn title="concept-serialize-xml-attributes">XML serialization of the attributes</dfn>
-            of an <code><a title="concept-element" data-spec="DOM4" class="externalDFN">Element</a></code>
-            <var>element</var> together with a <a title="concept-namespace-prefix-map">namespace prefix
-            map</a> <var>map</var>, a <a title="concept-generated-prefix">generated prefix index</a>
-            <var>prefix index</var> reference, a flag <var>ignore namespace definition attribute</var>, a
-            <var>duplicate prefix definition</var> value, and a flag <var>require well-formed</var>,
-            is the result of the following algorithm:
-        <ol>
-            <li>Let <var>result</var> be the empty string.
-            <li>Let <var>localname set</var> be a new empty
-                <dfn title="concept-namespace-localname-set">namespace localname set</dfn>. This <var>localname
-                set</var> will contain tuples of unique attribute
-                <code><a title="dom-attr-namespaceuri" data-spec="DOM4" class="externalDFN">namespaceURI</a></code>
-                and <code><a title="dom-element-localname" data-spec="DOM4" class="externalDFN">localName</a></code>
-                pairs, and is po
```

Comment:
This&nbsp;section should&nbsp;contain a&nbsp;`<dfn>` for&nbsp;`[CEReaction]`, which&nbsp;I’m&nbsp;adding in&nbsp;<https://github.com/w3c/DOM-Parsing/pull/57>.